app/views/curation_concerns/base/_show_actions.html.erb

```diff
@@ -1,7 +1,10 @@
 <div class="form-actions">
   <% if Sufia.config.analytics %>
-    <%= link_to "Analytics", sufia.stats_work_path(@presenter), id: 'stats', class: 'btn btn-default' %>
-  <% end %>
+    <% if @presenter.model_name.name ==  "GenericWork" %>
+      <%= link_to "Analytics", sufia.stats_work_path(@presenter), id: 'stats', class: 'btn btn-default' %>
```

Comment:
I might like to see this code in the presenter instead of the view. So the view would just have:

```ruby
<%= link_to "Analytics", @presenter.stats_path, id: 'stats', class: 'btn btn-default' %>
```

And the presenter would have, e.g.:

```ruby
def stats_path
  case model_name.name
  when 'GenericWork'
    sufia.stats_work_path(self)
  when 'FileSet'
    sufia.stats_file_path(self)
  else
    raise 'handle this better'
  end
end
```